src/goto-instrument/full_slicer.cpp

```diff
@@ -400,6 +429,9 @@ void full_slicert::operator()(
   Forall_goto_functions(f_it, goto_functions)
     if(f_it->second.body_available())
     {
+   	  if(f_it->first=="pthread_create")
+   		throw "--full-slice does not support C/Pthreads yet";
+
```

Comment:
This was fixed in  https://github.com/lucasccordeiro/cbmc/tree/dependence-graph-fix